### PR TITLE
German translation for Translation support PR

### PR DESCRIPTION
**Pull request by StealWonders**
This pull request adds German message translations to Velocity.
This requires https://github.com/VelocityPowered/Velocity/pull/230 beforehand.

### CODE REVIEW

File: proxy/src/main/resources/translations/de_de.json [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  "velocity.no-proxy-behind-proxy": {
+    "text": "Velocity hinter Velocity zu betreiben wird nicht unterstüzt.",
+    "color": "red"
+  },
+  "velocity.no-available-servers": {
+    "text": "Keine verfügbaren Server.",
+    "color": "red"
+  },
+  "velocity.already-connected": {
+    "text": "Du bist bereits mit diesem Proxy verbunden!",
```
Comment (Gabik21) [inline proxy/src/main/resources/translations/de_de.json]: The genus of proxy is masculine, therefore it should be 'dieser Proxy'.

File: proxy/src/main/resources/translations/de_de.json [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    "color": "red"
+  },
+  "velocity.moved-to-new-server": {
+    "text": "Du wurdest von dem Server auf dem Du warst gekickt.",
```
Comment (Gabik21) [inline proxy/src/main/resources/translations/de_de.json]: There are commas missing: 'Du wurdest von dem Server, auf dem Du warst, gekickt.'
Furthermore, I would suggest to rather use this translation: 'Du wurdest von dem Server, auf dem Du vorher warst, geworfen.'

File: proxy/src/main/resources/translations/de_de.json [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+  "velocity.command.not-found": {
+    "text": "Unbekannter Befehl.",
+    "color": "red"
+  },
+  "velocity.command.too-many-arguments": {
+    "text": "Zu viele Argumente.",
+    "color": "red"
+  },
+  "velocity.command.server-doesnt-exist": {
+    "text": "Der Server %s existert nicht.",
```
Comment (Gabik21) [inline proxy/src/main/resources/translations/de_de.json]: There is a letter missing in the word 'existiert'

**Comment by Gabik21** [comment]
You can also rebase on [this branch](/velocitypowered/velocity/tree/feature/translation) now :)

File: proxy/src/main/resources/translations/de_de.json (hunk 1 above)
Comment (StealWonders) [reply proxy/src/main/resources/translations/de_de.json]: Diesem ist correct in this case :)

File: proxy/src/main/resources/translations/de_de.json (hunk 2 above)
Comment (StealWonders) [reply proxy/src/main/resources/translations/de_de.json]: Yours sounds a bit better, thanks +1

File: proxy/src/main/resources/translations/de_de.json (hunk 1 above)
Comment (Gabik21) [reply proxy/src/main/resources/translations/de_de.json]: Whoops my bad

**Comment by astei** [comment]
Can you rebase on the `feature/translation` branch, please?

**Comment by StealWonders** [comment]
Did I do that correctly @astei ?

**Comment by Gabik21** [comment]
Rebase and merge are different actions. With a rebase your commits will be put on top of a certain branch. After that you'll need to do a force push because the history will be rewritten. Also on GitHub you want to change your target branch to feature/translation.

File: proxy/src/main/resources/translations/de_de.json [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+        "color": "gray"
+      }
+    ]
+  },
+  "velocity.no-proxy-behind-proxy": {
+    "text": "Velocity hinter Velocity zu betreiben wird nicht unterstüzt.",
+    "color": "red"
+  },
+  "velocity.no-available-servers": {
+    "text": "Keine verfügbaren Server.",
```
Comment (derklaro) [inline proxy/src/main/resources/translations/de_de.json]: In this case "Es ist kein Server verfügbar." is better :)

File: proxy/src/main/resources/translations/de_de.json (hunk 2 above)
Comment (derklaro) [inline proxy/src/main/resources/translations/de_de.json]: It's "Du wurdest von dem Server, auf dem Du warst gekickt."

File: proxy/src/main/resources/translations/de_de.json (hunk 4 above)
Comment (StealWonders) [reply proxy/src/main/resources/translations/de_de.json]: Stimme Dir zu :)

**Comment by astei** [comment]
I am closing this PR. If you are interested, then please retranslate the messages on the `dev/2.0.0` and open a new PR.

**Pull request closed without merging**